flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/StreamTask.java

```diff
@@ -128,7 +130,7 @@
 	protected Operator headOperator;
 
 	/** The chain of operators executed by this task */
-	private OperatorChain<OUT> operatorChain;
+	OperatorChain<OUT> operatorChain;
```

Comment:
this is deliberately private. I dont see it being used in package access so it can be reverted